Azure-RMSDocs/rms-client/clientv2-admin-guide-customizations.md

```diff
@@ -1015,7 +1015,10 @@ To modify this timeout, create the following advanced setting for the selected p
 Example PowerShell command, where your label policy is named "Global":
 
 ```PowerShell
-Set-LabelPolicy -Identity Global -AdvancedSettings @{EnableOutlookDistributionListExpansion="true"} @{OutlookGetEmailAddressesTimeOutMSProperty="3000"}
+Set-LabelPolicy -Identity Global -AdvancedSettings @{
+  EnableOutlookDistributionListExpansion="true"
```

Comment:
I see an attempt to unify the 2 settings. Shouldn't there be a comma at the end of this line??

Reply:
@bb-froggy can you take a look at @shsagir 's comment before we publish?

Reply:
It works without comma, this is the default syntax, see here: https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-hashtable?view=powershell-7.1#creating-hashtables-with-values

I don't know whether it would also work without them. The original line of code currently in the docs throws an error, though.

Reply:
thanks @bb-froggy ! I'll allow it - pushing it through in a separate PR. thanks for your contribution to docs!